Update versioning guidelines in navnestandard.qmd

aecorn opened the pull request:
Added a warning about the possibility of storing unversioned files alongside versioned ones, including potential pitfalls. Clarified the exceptions to versioning, specifying that only temporary data does not require versioning.

🤖 Automatisk beskjed 🤖

Har du husket å lese gjennom [retningslinjene for å bidra til manualen](https://manual.dapla.ssb.no/statistikkere/appendix/contribution.html)?
👉 Husk å skrive et nyhetsinnlegg dersom endringene er omfattende!

<!-- Reviewable:start -->
- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/statisticsnorway/dapla-manual/478)
<!-- Reviewable:end -->

aecorn committed:
Update versioning guidelines in navnestandard.qmd

Added a warning about the possibility of storing unversioned files alongside versioned ones, including potential pitfalls. Clarified the exceptions to versioning, specifying that only temporary data does not require versioning.

snergaard committed:
Endringer i språket

alexcroz commented:
God job, but I think we should remove any reference to unversioned files completely. I did not realise unversioned files were still mentioned in the docs.

If we mention this method of versioning we should discourage its use more explicitly and point to paths in ssb-fagfunksjoner.

aecorn commented:
This stems from a discussion with @BjornRoarJoneid, @sindre-haugen among others.

I think this unversioned business is very important for some users. But I'm completely on-board that it's probably a bad idea, and it's easy to see problems on the horizon stemming from this decision. I guess before removing the option, we need to contact the users for whom this was important, and try to have the discussion again. Explaining how it can be remedied with automatic version picking and logging.

BjornRoarJoneid commented on dapla-manual/statistikkere/navnestandard.qmd:
```diff
@@ -211,14 +211,21 @@ ssb-dapla-example-data-produkt-prod/
+1. Hvordan sikrer man reproduserbarhet, når det ikke logges hos konsumenten hvilken versjon av filen som ble konsumert?
+2. Kan du garantere at du alltid skriver over den uversjonerte filen, eller er det en mulighet for at den kan bli "hengende bak" de versjonerte filene?
+:::
+
```
@aecorn I think we should also add the following warning in the "callout-warning":

**Det anbefales derfor kun å benytte filnavn med versjonsnummer (unngå uversjonerte filer)! Dette vil også forenkle dokumentasjon av datasett i DataDoc, tilgjengeliggjøring i datakatalogen, og er nødvendig for en ryddig deling av data.**

aecorn replied:
Ive added the change to the branch and PR

BjornRoarJoneid reviewed (changes requested):
@aecorn Suggestion for an additional warning in the text.

aecorn committed:
Recommend using versioned filenames for data files

Added recommendation to use versioned filenames only to ensure data consistency and simplify documentation.

aecorn committed:
Update navnestandard.qmd
